src/main.py

```diff
@@ -22,40 +22,46 @@ def message(sid, data):
 @sio.event
 def send_video_event(sid, data):
     print(sid, data)
-    sio.emit(event="receive_video_event", data=data['eventInfo'], room=data['myRoomId'], skip_sid=sid)
+    sio.emit(event="receive_video_event", data=data['eventInfo'], room=data['roomId'], skip_sid=sid)
     return "OK"
 
 @sio.event
 def join_room(sid, roomId):
     print(sid, roomId)
-    if(sio.manager.rooms.get("/").get(roomId) is None):
+    if not roomId in room_dict:
         return "ROOM_NOT_FOUND"    
     sio.enter_room(sid, roomId)
-    user_room_dict[sid] = {
-        'room': roomId
+    room_dict[roomId] = {
+        sid: {}
     }
-    user_room_dict[sid]['permissions'] = PERMISSIONS_USER.copy()
-    sio.emit(event='permissions', data={'permissions': user_room_dict[sid]['permissions']}, to=sid)
+    room_dict[roomId][sid]['permissions'] = PERMISSIONS_USER.copy()
+    sio.emit(event='permissions', data={
+        'permissions': room_dict[roomId][sid]['permissions'],
+        'roomId': roomId,
+        })
```

Comment:
Here should be `to=sid`, so it won't be sent to all users connected to the server.